Kitodo/src/main/java/org/kitodo/production/forms/ProcessForm.java

```diff
@@ -300,14 +300,16 @@ private void deleteProcess(Process processToDelete) {
             processToDelete.setProject(null);
             processToDelete.getTemplate().getProcesses().remove(processToDelete);
             processToDelete.setTemplate(null);
+            ServiceManager.getCommentService().removeAll(
```

Comment:
Why don't you use 
`processToDelete.getComments()` here?

Reply:
It turns out that my fix was more complex than necessary. I removed this part.